resources/mysqlbinlog/mysqlbinlog_8.0.28-macos11-arm64_test.go

```diff
@@ -0,0 +1,43 @@
+//go:build darwin && arm64
+// +build darwin,arm64
+
+package mysqlbinlog
+
+import (
+	"embed"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/bytebase/bytebase/resources/utils"
+)
+
+//go:embed mysqlbinlog-8.0.28-macos11-arm64.tar.gz
+var resources embed.FS
+
+// TestBootMySQLBinlog tests whether mysqlbinlog can be started on the target platform
+// to check whether the lib extraction is correct.
+func TestBootMySQLBinlog(t *testing.T) {
+	tarName := "mysqlbinlog-8.0.28-macos11-arm64.tar.gz"
+	version := "mysqlbinlog-8.0.28-macos11-arm64"
+	tarF, err := resources.Open(tarName)
+	if err != nil {
```

Comment:
Please check no errors like this
https://github.com/bytebase/bytebase/blob/6e3648211e8f8c81cabfe4aedc7b6175c1f2c060/tests/schema_update_test.go#L25